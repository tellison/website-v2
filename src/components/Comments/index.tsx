import React from 'react';

const Comments = () => {
  return (
    <p
      style={{
        alignItems: 'center',
        display: 'flex',
        flexWrap: 'wrap',
        justifyContent: 'flex-start',
        backgroundColor: '#e3e3e3',
        borderRadius: '4px',
        padding: '0.5em',
        marginTop: '1em',
      }}
    >
      <em>Do you have questions or want to discuss this post? Hit us up on the <a href='https://adoptium.slack.com/'>Adoptium Slack workspace</a>!</em>
    </p>
  );
};

export default Comments;
