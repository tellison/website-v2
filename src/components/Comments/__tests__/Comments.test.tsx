import React from 'react';
import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest'
import Comments from '../index';

describe('Comments component', () => {
    it('should render correctly', () => {
        const { container } = render(<Comments />);

        expect(container).toMatchSnapshot();
    });
});