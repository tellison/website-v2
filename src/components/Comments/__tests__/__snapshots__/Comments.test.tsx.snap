// Vitest Snapshot v1

exports[`Comments component > should render correctly 1`] = `
<div>
  <p
    style="align-items: center; display: flex; flex-wrap: wrap; justify-content: flex-start; background-color: rgb(227, 227, 227); border-radius: 4px; padding: 0.5em; margin-top: 1em;"
  >
    <em>
      Do you have questions or want to discuss this post? Hit us up on the 
      <a
        href="https://adoptium.slack.com/"
      >
        Adoptium Slack workspace
      </a>
      !
    </em>
  </p>
</div>
`;
